atc/db/lock/lock_test.go

```diff
@@ -104,6 +104,36 @@ var _ = Describe("Locks", func() {
 			Expect(acquired).To(BeFalse())
 		})
 
+		It("Acquire times out acquiring the mutex lock", func() {
+
+			done := make(chan struct{})
+			readyToAcquire := make(chan struct{})
+
+			fakeLockDB := new(lockfakes.FakeLockDB)
+			factory := lock.NewTestLockFactory(fakeLockDB)
+
+			fakeLockDB.AcquireStub = func(id lock.LockID) (bool, error) {
+				select {
+				case <-time.After(time.Minute):
```

Comment:
If default timeout is 1 second, should we make this test to a bit closer to that so if someone extend the default timeout this test could protect that? How about `time.Minute * 2`

Reply:
Updated